opensearch_py_ml/ml_commons/ml_commons_client.py

```diff
@@ -63,6 +63,76 @@ def upload_model(
             model_path, model_config_path, isVerbose
         )
 
+    def upload_pretrained_model(
+        self, name: str, version: str, model_format: str, load_model: bool = True
+    ):
+        """
+        This method uploads a pretrained model into opensearch cluster using ml-common plugin's api.
+        First, this method creates a model id to store model info
+        and then loads the model from Hugging Face if load_model is True.
```

Comment:
then loads the model in memory. We don't load model from huggingFace. Model is downloaded in their index and then it will be loaded into memory using DJL.